src/uproot/source/http.py

```diff
@@ -351,7 +353,7 @@ def task(resource):
     )
     _content_range = re.compile(b"Content-Range: bytes ([0-9]+-[0-9]+)", re.I)
 
-    def is_multipart_supported(self, ranges, response):
+    def is_multipart_supported(self, ranges: list[(int, int)], response) -> bool:
```

Comment:
It's an [HTTPResponse](https://docs.python.org/3/library/http.client.html#httpresponse-objects).

```suggestion
    def is_multipart_supported(self, ranges: list[(int, int)], response: http.client.HTTPResponse) -> bool:
```